### PR TITLE
Jsoup version bump

### PR DESCRIPTION
**Pull request by thisaltennakoon**
Fix https://github.com/wso2/api-manager/issues/1261

### CODE REVIEW

**Review by npamudika — APPROVED** [review]

**Review by RakhithaRR — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -321,6 +321,11 @@
                 <artifactId>antlr4-runtime</artifactId>
                 <version>${version.antlr.runtime}</version>
             </dependency>
+            <dependency>
```
Comment (SanojPunchihewa) [inline pom.xml:324]: Let's move this to the `monitoring-dashboard/pom.xml` module since this is only used there

Comment (thisaltennakoon) [reply pom.xml]: Fixed in https://github.com/wso2/product-mi-tooling/pull/329/commits/05d0ab83eac91e980e3de9540105565c601ee04a

Comment (SanojPunchihewa) [reply pom.xml]: Shall we define the jsoup version as a property?

Comment (thisaltennakoon) [reply pom.xml]: Fixed in https://github.com/wso2/product-mi-tooling/pull/329/commits/21cc3693c4b3d91d7245ed8e43c33d2cceb95955

**Review by SanojPunchihewa — APPROVED** [review]

**Review by npamudika — APPROVED** [review]

**Pull request merged**